StaticInstaller.php

```diff
@@ -111,10 +126,31 @@ public static function composerFinishUpdate(Event $composer_event = null)
                 $text .= '-> Updating app "' . $app_alias . '": ' . ($result ? $result : 'Nothing to do') . ".\n";
                 self::printToStdout($text);
             }
-            unset($temp['update']);
+        }
+        if ($temp['update']!=null){
+            $updatedPackages = $temp['update'];
+        }
+        self::printToStdout("Unlink unused backup components:\n");
+        $composerContent = file_get_contents('composer.json');
+        $obj = json_decode($composerContent);
+        $backupTime = $temp['backupTime'];
+        $installer = new self();
+        $unlinkResult = array();
+        foreach($obj->require as $requiredPackage => $packageInfo){
```

Comment:
Hier würdest du alle Packages bekommen, die in der Root-Composer.json der Installation required sind. Was ist mit den Dependencies? Wäre es nicht viel einfacher alle Ordner im Backup durchzugehen und zu schauen, ob diese in der install.temp.json vorhanden sind? Man könnte sogar diese install.temp.json in den Backup-Ordner kopieren (natürlich nicht als temp dann) - um die Nachvollziehbarkeit zu erhöhen.